octree_creation_app-assets/uijson/octree_mesh.ui.json

```
@@ -26,7 +26,10 @@
             "{b99bd6e5-4fe1-45a5-bd2f-75fc31f91b38}",
             "{0b639533-f35b-44d8-92a8-f70ecff3fd26}",
             "{9b08bb5a-300c-48fe-9007-d206f971ea92}",
-            "{19730589-fd28-4649-9de0-ad47249d9aba}"
+            "{19730589-fd28-4649-9de0-ad47249d9aba}",
+            "{b3a47539-0301-4b27-922e-1dde9d882c60}",
+            "{a81c6b0a-f290-4bc8-b72d-60e59964bfe8}",
+            "{41018a45-01a0-4c61-a7cb-9f32d8159df4}"
```

Missing a couple here:
```suggestion
            "{41018a45-01a0-4c61-a7cb-9f32d8159df4}",
            "{deebe11a-b57b-4a03-99d6-8f27b25eb2a8}",
            "{275ecee9-9c24-4378-bf94-65f3c5fbe163}"
```